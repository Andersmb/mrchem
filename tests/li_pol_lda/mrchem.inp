# vim:syntax=sh:

world_prec = 1.0e-3               # Overall relative precision
world_size = 5                    # Size of simulation box 2^n

MPI {
  numerically_exact = true        # Guarantee identical results in MPI
}

Molecule {
  multiplicity = 2
$coords
Li      0.0000  0.0000  0.0000
$end
}

WaveFunction {
  method = LDA                    # Wave function method (HF or DFT)
  restricted = false
}

Properties {
  scf_energy = true               # Compute ground state energy
  polarizability = true
}

SCF {
  run = false
  initial_guess = SAD_DZ
}

Response {
  kain = 3                        # Length of KAIN iterative history
  max_iter = 10
  orbital_thrs = 1.0e-2           # Convergence threshold in orbital residual
  directions = [0,0,1]
}

Polarizability {
  frequency = [0.0]
}
