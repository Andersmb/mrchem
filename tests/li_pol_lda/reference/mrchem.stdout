                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                                                                ***
***          __  __ ____   ____ _                                  ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \               ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |              ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|              ***
***                                                                ***
***         VERSION    0.2.0        (rev.   81247d7b)              ***
***                                                                ***
***         Stig Rune Jensen   <stig.r.jensen@uit.no>              ***
***         Luca Frediani      <luca.frediani@uit.no>              ***
***         Peter Wind         <peter.wind@uit.no>                 ***
***                                                                ***
**********************************************************************

----------------------------------------------------------------------

 MPI processes         :                                            1
 OpenMP threads        :                                            1
 Total cores           :                                            1
                                                                      
----------------------------------------------------------------------

XCFun DFT library Copyright 2009-2011 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

----------------------------------------------------------------------

 MRCPP version         : 1.1.0
 Git revision          : 36888523

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : NONE

----------------------------------------------------------------------


                                                                      
======================================================================
                       MultiResolution Analysis
----------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
----------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                     Initializing Molecule                      ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            0
 Multiplicity          :                                            2
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0      Li          :       0.000000       0.000000       0.000000
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000       0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***              Computing Initial Guess Wavefunction              ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Diagonalize Hamiltonian matrix
 Precision             : 1.00000e-03
 Restricted            : False
 Hamiltonian           : Superposition of Atomic Densities (SAD)
 Functional            : LDA (SVWN5)
 AO basis              : Hydrogenic orbitals
 Zeta quality          : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
                          Molecular Orbitals
----------------------------------------------------------------------
 Alpha electrons       :                                            2
 Beta electrons        :                                            1
 Total electrons       :                                            3
----------------------------------------------------------------------
    n  Occ Spin        :                                         Norm
----------------------------------------------------------------------
    0    1    a        :                           1.000000000000e+00
    1    1    a        :                           9.999999447722e-01
    2    1    b        :                           1.000000000000e+00
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***              Computing Ground State Wavefunction               ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Compute initial energy
 Method                : DFT (SVWN5)
 Precision             : 1.00000e-03
 Localization          : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                7.917492190570
 E-N energy            :           (au)              -17.600794704106
 Coulomb energy        :           (au)                4.105234165256
 Exchange energy       :           (au)                0.000000000000
 X-C energy            :           (au)               -1.717998652012
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
----------------------------------------------------------------------
 Electronic energy     :           (au)               -7.296067000292
 Nuclear energy        :           (au)                0.000000000000
----------------------------------------------------------------------
 Total energy          :           (au)           -7.296067000292e+00
                       :     (kcal/mol)           -4.578351129142e+03
                       :       (kJ/mol)           -1.915582113676e+04
                       :           (eV)           -1.985360884955e+02
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***             Computing Linear Response Wavefunction             ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Static linear response
 Method                : DFT (SVWN5)
 Perturbation          : h_E_dip[z]
 Max iterations        : 10
 KAIN solver           : 3
 Localization          : Off
 Start precision       : 1.00000e-03
 Final precision       : 1.00000e-03
 Helmholtz precision   : 1.00000e-03
 Property threshold    : Off
 Orbital threshold     : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
 Iter         MO residual      Symmetric property              Update
----------------------------------------------------------------------
    0        1.000000e+00          0.000000000000        0.000000e+00
    1        1.179433e+01        -53.790213511671       -5.379021e+01
    2        6.322027e+00       -116.122330136539       -6.233212e+01
    3        1.539591e+00       -120.285459241636       -4.163129e+00
    4        1.181818e+00       -135.982272744216       -1.569681e+01
    5        3.464292e-01       -137.352895647902       -1.370623e+00
    6        1.612162e-01       -138.631855971302       -1.278960e+00
    7        5.890191e-02       -139.486608699907       -8.547527e-01
    8        1.300792e-02       -139.566251916183       -7.964322e-02
    9        3.801377e-03       -139.574487320761       -8.235405e-03
----------------------------------------------------------------------
                   SCF converged in 9 iterations!
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                 Printing Molecular Properties                  ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            0
 Multiplicity          :                                            2
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0      Li          :       0.000000       0.000000       0.000000
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000       0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


======================================================================
                           Orbital Energies
----------------------------------------------------------------------
    n  Occ Spin        :                                      Epsilon
----------------------------------------------------------------------
    0    1    a        :           (au)               -1.816789693174
    1    1    a        :           (au)               -0.113317396219
    2    1    b        :           (au)               -1.801617110695
----------------------------------------------------------------------
 Sum occupied          :           (au)               -3.731724200089
======================================================================


======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                7.917492190570
 E-N energy            :           (au)              -17.600794704106
 Coulomb energy        :           (au)                4.105234165256
 Exchange energy       :           (au)                0.000000000000
 X-C energy            :           (au)               -1.717998652012
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
----------------------------------------------------------------------
 Electronic energy     :           (au)               -7.296067000292
 Nuclear energy        :           (au)                0.000000000000
----------------------------------------------------------------------
 Total energy          :           (au)           -7.296067000292e+00
                       :     (kcal/mol)           -4.578351129142e+03
                       :       (kJ/mol)           -1.915582113676e+04
                       :           (eV)           -1.985360884955e+02
======================================================================


======================================================================
                            Polarizability
----------------------------------------------------------------------
 Frequency             :           (au)                      0.000000
                       :         (cm-1)                      0.000000
----------------------------------------------------------------------
 Total tensor          :       0.000000       0.000000       0.000000
                       :       0.000000       0.000000       0.000000
                       :      -0.000000      -0.000000     139.574487
 Isotropic average     :           (au)                     46.524829
======================================================================


                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                         Exiting MRChem                         ***
***                                                                ***
***                    Wall time :  0h  1m 40s                     ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
